tests/test_paperqa.py

```diff
@@ -196,6 +198,15 @@ def test_maybe_is_text() -> None:
     )
     assert maybe_is_text(r.text)
 
+    # Test Japanese Wikipedia page for multilingual support
+    r_ja = httpx.get(
+        "https://ja.wikipedia.org/wiki/%E6%97%A5%E6%9C%AC",
+        headers={
+            "User-Agent": "PaperQA testing (https://github.com/Future-House/paper-qa)"
+        },
+    )
```

Comment:
Since you have added an HTTP request, can you refresh the HTTP cache here:

```bash
rm tests/cassettes/test_maybe_is_text.yaml
```

Then re-run the test to get the cassette, and make sure `prettier` cleans up the cassette